restart.sh

```diff
@@ -1,2 +1,10 @@
+echo "Pulling fresh changes from GitHub"
 git pull
+
+echo "Restarting daemon, gunicorn, nginx and socket"
+sudo systemctl daemon-reload
+sudo systemctl restart gunicorn
 sudo systemctl restart gunicorn.socket
+sudo nginx -t && sudo systemctl restart nginx
```

Comment:
If things look broken, it could be a good idea to just restart everything, to pick up any changes that might not have been caught. 

These are the specific changes for restarting service:
- `daemon-reload`: if there has been a change to the `/etc/systemd/system/gunicorn.service`. Will usually need to restart the server after as well. 
- `restart gunicorn`: same as above, after the daemon has been reloaded. If you move the `phsite.sock` socket, you'll need to find it, and delete it, should probably also just `killall gunicorn`, then run `sudo systemctl start gunicorn && sudo systemctl enable gunicorn`
- `restart gunicorn.socket`: if changes are made to `/etc/systemd/system/gunicorn.socket`. This is where gunicorn looks for the socket to listen on. 
- `nginx`: changes to the domain name, certbort (Let's encrypt), or other changes to `/etc/nginx/sites-available/phsite`